src/config.js

```diff
@@ -223,6 +223,7 @@ export const BZ_QUERIES = {
       },
   },
 };
+export const bugzillaAPILink = 'http://localhost:2000/api/';
```

Comment:
As this isn't really a bugzilla API but more a wrapper/helper, please rename it and/or add more comments :)